### ISSUE TITLE
please revert to old option

### ISSUE THREAD

**Issue by tkl45** [dobbelina/repository.dobbelina#1395]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.

### Description

after clicking a site i used to be able to switch to 'shift' and it would display each video in landscape mode. at some point it changed and now displays a cropped version of portrait mode. can you please change it back? i wish all the display modes were in landscape mode. to be able to see it uncropped. (maybe a little larger with the shift mode displays as well.) otherwise thank you for cumination its been my go to for along time now!

**Comment by 12asdfg12** [comment]
Change the content type in settings.

**Comment by tkl45** [comment]
toggling from movies to video does not work.  shift and poster are are in (cropped) portrait mode. the other modes that do display the landscaped full screen have a gradient black bottom hindering the video thumbnail..  i am using firestick

**Comment by 12asdfg12** [comment]
the pictures are landscape and not cropped, just like on the website

![image](https://github.com/user-attachments/assets/821afd2c-e6b9-4733-be18-596df0188c35)

**Comment by tkl45** [comment]
![20240728_133755](https://github.com/user-attachments/assets/2638be68-fc2c-4552-b354-01aa13b174fb)
 ..not sure why mine is different

**Comment by 12asdfg12** [comment]
you are right, in kodi 21 it looks like in your picture, but in kodi 19 and 20 it is ok.
Something has probably changed in the Kodi skin.
